docs/community/workshops.md

```diff
@@ -0,0 +1,39 @@
+# Workshops
+Find us at the following upcoming and past workshops!
+
+## DataJoint Office Hours
+
+The DataJoint open source team will be offering monthly Office Hours.  These are public workshops on Zoom where we will offer two half-hour sessions of free hands-on support to teams using DataJoint software.
+
+For each half-hour session, a team can sign up to work through their specific question.  Teams who wish to receive this support can either share their workflow directly or work with DataJoint engineers to replicate the issue in a test environment.   To be fair to the community, no team will be selected for support more than once per quarter.
+
+[Sign up for Office Hours!](https://docs.google.com/forms/d/e/1FAIpQLSeMhZtzQQWB47I8HfPcJ5_pFyMhZO284PLIblDfshe30dEuXw/viewform)
+
+## Upcoming workshops
+
++ NIH BRAIN Initiative Meeting - June 21-22, 2022
+
++ INCF Assembly - September 12-16, 2022
+
++ Society for Neuroscience - November 12-16, 2022
+
+## Past workshops
+
++ UCL Neuropixels Course - October 19, 2021
+
++ INCF Neuroinformatics Training Week - August 30 - September 2, 2021
+     + [Session 1 Recording](https://vimeo.com/586441656)
```

Comment:
When do we want to merge this? Before Monday? If after is ok, I suggest swapping for our youtube links to better equip us to monitor traffic